lib/Slider/Slider.js

```diff
@@ -898,6 +898,7 @@ module.exports = kind(
 	*/
 	previous: function () {
 		this.set('value', this.value - this._jumpIncrementAmount);
+		if (this.popup && this.$.popup.getShowing()) return true;
```

Comment:
Ah ha, I was unclear on the UX and whether we wanted the popup to display at all when pressing the jump increment buttons. If that is the case, should the popup be displayed when you 5-way hold on one of the jump increment buttons? Currently with this change, the popup only displays after the 5-way key is released.